src/main/java/com/google/flourbot/execution/MacroExecutionModuleImplementation.java

```diff
@@ -77,6 +77,10 @@ public String execute(String userEmail, String message, String threadId) throws
       case SHEET_APPEND:
         // Read instructions on what to write
 
+        // Remove "@Macrobot macroName" from message
+        String[] messages = message.split(" ", 3);
```

Comment:
That's a good start but we are missing:
- error handling, what if the message contains less than 3 sections, what if the user made a mistake and used two spaces etc.
- two sections are possible if the message is missing the macro name (i.e. the cached value is used).

Reply:
Feel free to submit and we can come back to the details later if there is time.

Reply:
There might be " " in the user's message. Ie "@Marcobot macroName Good Morning" becomes ["@MacroBot", "macroName", "Good", "Morning"] 

In this case, the array should be sliced from [2 : length], and rejoined with " ".

Reply:
@tsa87 I believe the split I have will still work (take a look at this screenshot: https://screenshot.googleplex.com/wGk1Qj8FUhb.png). However, the bot's name will have to be one word without spaces, so we should maybe specify that in the website.

Reply:
@lipczak-g I have added an error check and an if statement to split the message differently depending on whether or not the message contains the macroname.

Reply:
Okay great! If you have a forloop, that works.